module/item/item-attack.mjs

```diff
@@ -116,13 +116,12 @@ export async function AttackOptions(item) {
 }
 
 export async function _processAttackOptions(item, formData) {
+    if (item.getAoeModifier()) {
+        await AttackAoeToHit(item, formData);
```

Comment:
I was a little iffy about this change (admittedly a cleanup). it's await and return are a peculiar combination and since you have brought it up I'm tempted to just revert if I can't come to an understanding of the process.

Reply:
re-read your comment and realise now what you meant. Will fix

Reply:
If you're uncomfortable with a `return await AttackAoeToHit...` you could do:

```javascript
await AttackAoeToHit ...
return;
```